### PR TITLE
feat(@spartacus/storefrontapp): translation files

### LINKED ISSUE #1487: 06a Extract wordings to translation files
It's the master ticket for extracting hardcoded wordings from templates and the code to translation files.

We should create detailed followup tickets when we have suggested strucutre of translation files as a result of spikes.

### PR DESCRIPTION
**Pull request by Pucek9**
Closes #1487

### CODE REVIEW

File: projects/storefrontapp/src/assets/i18n/order-history.en.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "labels": {
+    "orderHistory": "Order history",
+    "order": "Order #",
+    "placed": "Date",
+    "status": "Status",
+    "shipped": "Total",
+    "shippedOn": "We have no order records for this account."
```
Comment (Platonn) [inline projects/storefrontapp/src/assets/i18n/order-history.en.json]: The value doesn't match the key. Probably it's a mistake.

File: projects/storefrontapp/src/assets/i18n/register.en.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    "firstName": "First Name",
+    "lastName": "Last Name",
+    "emailAddress": "Email Address",
+    "password": "Password"
+  },
+  "validations": {
+    "passwordMinRequirements": "Password must be six characters minimum, with one uppercase letter, one number, one symbol",
+    "bothPasswordMustMatch": "Both password must match"
+  },
+  "conditions": {
```
Comment (Platonn) [inline projects/storefrontapp/src/assets/i18n/register.en.json]: what about `consents`?

File: projects/storefrontapp/src/assets/i18n/register.en.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    "useMyPersonalDataToReceiveEmailNewslettersFor": "Use my personal data to receive e-mail newsletters for marketing campaigns. To change your settings, go to Consent Management in My Account.",
```
Comment (Platonn) [inline projects/storefrontapp/src/assets/i18n/register.en.json]: can we simplify this key? for example. `emailMarketing`?

File: projects/storefrontapp/src/assets/i18n/register.en.json [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    "iAmConfirmingThatIHaveReadAndAgreedWith": "I am confirming that I have read and agreed with the"
```
Comment (Platonn) [inline projects/storefrontapp/src/assets/i18n/register.en.json]: 1. Can we simplify this key?
2. It's also a bit inconsistent with `checkout-review.en.json`

File: projects/storefrontapp/src/assets/i18n/payment-review.en.json [hunk 5]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (Platonn) [inline projects/storefrontapp/src/assets/i18n/payment-review.en.json]: As discussed offline, this file should be rather something like `order-summary` or `order-details`. 
And current `order-summary` should become something like `order-cost` or `order-value`

File: projects/storefrontapp/src/assets/i18n/order-details.en.json [hunk 6]
```diff
@@ -0,0 +1,10 @@
+{
+  "labels": {
+    "order": "Order #",
+    "placed": "Placed",
+    "status": "Status",
+    "shipped": "SHIPPED",
```
[the remark this answers is not in the document]
Comment (Pucek9) [reply projects/storefrontapp/src/assets/i18n/order-details.en.json]: SHIPPED is probably provided by backend so I'll remove it

File: projects/storefrontapp/src/assets/i18n/address.en.json [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+    "zipCode": "Zip code",
+    "phoneNumber": "Phone number (optional)"
+  },
+  "actions": {
+    "saveAsDefault": "Save as default",
+    "chooseAddress": "Choose address",
+    "continue": "Continue",
+    "setAsDefault": "set as default",
+    "backToAddressList": "Back to address list",
+    "addAddress": "Add address",
```
[the remark this answers is not in the document]
Comment (Pucek9) [reply projects/storefrontapp/src/assets/i18n/address.en.json]: That's not same. Add new address is displayed on addresses list, and Add address means "Submit" for form.

File: projects/storefrontapp/src/assets/i18n/payment-review.en.json (hunk 5 above)
Comment (Pucek9) [reply projects/storefrontapp/src/assets/i18n/payment-review.en.json]: why not order-review?

File: projects/storefrontapp/src/assets/i18n/order-history.en.json [hunk 8]
```diff
@@ -1,11 +1,11 @@
...
   "labels": {
     "orderHistory": "Order history",
-    "order": "Order #",
+    "orderId": "Order #",
     "placed": "Date",
     "status": "Status",
-    "shipped": "Total",
-    "shippedOn": "We have no order records for this account."
+    "total": "Total",
+    "weHaveNoOrderRecordForThisAccount": "We have no order records for this account."
```
Comment (Platonn) [inline projects/storefrontapp/src/assets/i18n/order-history.en.json]: Can we simpify it to `noOrders`?

File: projects/storefrontapp/src/assets/i18n/register.en.json [hunk 9]
```diff
@@ -20,14 +19,15 @@
...
+    "confirmPassword": "Confirm Password"
   },
   "validations": {
     "passwordMinRequirements": "Password must be six characters minimum, with one uppercase letter, one number, one symbol",
     "bothPasswordMustMatch": "Both password must match"
   },
-  "conditions": {
-    "useMyPersonalDataToReceiveEmailNewslettersFor": "Use my personal data to receive e-mail newsletters for marketing campaigns. To change your settings, go to Consent Management in My Account.",
-    "iAmConfirmingThatIHaveReadAndAgreedWith": "I am confirming that I have read and agreed with the"
+  "consents": {
```
Comment (Platonn) [inline projects/storefrontapp/src/assets/i18n/register.en.json]: For simplicity of the structure (not to introduce new key), let's move it to `labels`:
- `emailMarketingConsent`
- `confirmThatRead`

Comment (Pucek9) [reply projects/storefrontapp/src/assets/i18n/register.en.json]: I separated it because these are descriptions of checkboxes and not individual labels

**Pull request merged**